.dockerignore

```diff
@@ -0,0 +1,3 @@
+.venv
```

Comment:
I've personally used this dockerignore for a long time, do you see any issue with ignoring more files like this?

```
*
!/README.md
!/pyproject.toml
!/src/**
```